shell/platform/darwin/ios/framework/Source/SemanticsObject.mm

```diff
@@ -284,7 +284,7 @@ - (BOOL)isAccessibilityElement {
   // item that is currently off screen but the a11y navigation needs to know
   // about.
   return (([self node].flags & ~flutter::kScrollableSemanticsFlags) != 0 &&
-          [self node].flags != static_cast<int32_t>(flutter::SemanticsFlags::kIsHidden)) ||
+          ([self node].flags & ~static_cast<int32_t>(flutter::SemanticsFlags::kIsHidden)) != 1) ||
```

Comment:
This doesn't look correct, why are you checking against the 1 value?  Are you trying to check to see if the kIsHidden bit is turned on?  It should look like what is above in that case: `([self node].flags & flutter::kScrollableSemanticsFlags) != 0`

Checking anything against 1 is going to be fragile since it would depend on which bit is kIsHidden.

Reply:
oops, I made a mistake, I surprised this still work